src/main/kotlin/frc/robot/robotstate/RobotStates.kt

```diff
@@ -0,0 +1,8 @@
+package frc.robot.robotstate
+
+enum class ROBOT_STATE() {
+    Idling,
+    Intaking,
+    Shooting
+}
```

Comment:
The constant names can be left lowercase, but the class name is a regular class name. From the docs:
> For enum constants, it's OK to use either all uppercase, underscore-separated ([screaming snake case](https://en.wikipedia.org/wiki/Snake_case)) names (`enum class Color { RED, GREEN }`) or upper camel case names, depending on the usage.

An example from the docs: 
```kt
enum class Direction {
    NORTH,
    SOUTH,
    WEST,
    EAST, // trailing comma
}
```
So this is fine:
```suggestion
enum class RobotState() {
    Idle,
    Intaking,
    Shooting
}
```
Although I personally prefer this:

```suggestion
enum class RobotState() {
    IDLE,
    INTAKING,
    SHOOTING
}
```
Note that the more common term in this context is "idle" and not "idling".

Reply:
I prefer Idling, so that's what i told him to call it

Reply:
I'd resolve this but the class name still needs to be changed. I put my sources on the other conversation.